src/mumble/Themes.cpp

```diff
@@ -105,6 +147,30 @@ bool Themes::apply() {
 	return result;
 }
 
+bool Themes::detectSystemTheme() {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
+	return false; // This should not be called for Qt 6.5 and above
```

Comment:
Instead of returning ``false`` here, you can use the ``QGuiApplication::styleHints()->colorScheme();`` right here.  This would be even better as you could skip the Qt Version check in ``Themes::applyConfigured()`` entirely.

``return QGuiApplication::styleHints()->colorScheme() == Qt::ColorScheme::Dark``